src/components/BettingChart.js

```diff
@@ -1,62 +1,94 @@
-import React from 'react';
-import { Query } from 'react-apollo';
-import { BETS_QUERY } from '../graphql/queries';
+import React, { Component } from "react";
+import { Query } from "react-apollo";
+import { BETS_QUERY, BETS_SUBSCRIPTION } from "../graphql/queries";
 
-const BettingChart = () => (
-  <Query query={BETS_QUERY}>
-    {({ loading, error, data }) => {
-      if (loading) return <div>Fetching</div>;
-      if (error) return <div>Error</div>;
+class BettingChart extends Component {
+  subscribeToMoreBets = subscribeToMore => {
+    subscribeToMore({
+      document: BETS_SUBSCRIPTION,
+      updateQuery: (prev, { subscriptionData }) => {
+        if (!subscriptionData.data) return prev;
+        const newBet = subscriptionData.data.betAdded;
+        const exists = prev.bets.find(({ id }) => id === newBet.id);
+        if (exists) return prev;
+        return Object.assign({}, prev, {
```

Comment:
We need to implement a limit when adding to the bets array. At the moment the array size keeps increasing infinitely.